### DISCUSSION TITLE
Getting Started Guide Link in the Extension is Broken

### DISCUSSION THREAD

**Discussion by sogrady** [community/community#6829] [Copilot Conversations]
If you click the getting started guide from VS Code following installation of the extension, the link (https://github.com/github/copilot-docs/blob/main/docs/gettingstarted.md) is a 404.

**Comment by saakethkoka** [comment]
They updated it at a new url:
https://github.com/github/copilot-docs/blob/main/docs/jetbrains/gettingstarted.md

**Reply by andrewlayman** [reply]
The docs are here: https://github.com/github/copilot-docs/

**Comment by sogrady** [comment]
The issue is that the extension links to the old docs, not the new ones.

**Reply by andrewlayman** [reply]
Thank you.  We will look into fixing that.

**Comment by hsenag** [comment]
Apologies - this was a mistake on our side. The [old URL](https://github.com/github/copilot-docs/blob/main/docs/gettingstarted.md) is restored for now, and new versions of the plugin point to the [new URL](https://github.com/github/copilot-docs/tree/main/docs#getting-started).
